### ISSUE TITLE
Error "Errno::ENOENT - No such file or directory" - Bolts

### ISSUE THREAD

**Issue by marcelojunior7** [CocoaPods/CocoaPods#4189]
Report

What did you do?
Ran pod install on my project.

What did you expect to happen?
pods would update and I can build my project.

What happened instead?
I got the error below and pod install failed.

hockey/enterprise staging
   CocoaPods : 0.38.2
        Ruby : ruby 2.0.0p481 (2014-05-08 revision 45883) [universal.x86_64-darwin14]
    RubyGems : 2.4.5
        Host : Mac OS X 10.10.5 (14F27)
       Xcode : 6.4 (6E35b)
         Git : git version 2.1.3.36.g8e36a6d
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 2939f3c6220eb6bb75923e372290e191b77ec3ac
```

### Plugins

```
cocoapods-plugins : 0.4.2
cocoapods-stats   : 0.5.3
cocoapods-trunk   : 0.6.4
cocoapods-try     : 0.4.5
```

### Podfile

```ruby
# Uncomment this line to define a global platform for your project
# platform :ios, '6.0'
use_frameworks!

target 'HQKickStarter' do
pod 'AFNetworking', "~> 2.0"
pod 'YALSideMenu'
pod 'FBSDKCoreKit', "4.5.1"
pod 'FBSDKLoginKit', "4.5.1"
pod 'FBSDKShareKit', "4.5.1"
pod 'Bolts', "1.1"

end

target 'HQKickStarterTests' do

end
```

### Error

```
Errno::ENOENT - No such file or directory - /Users/marcelo/Desktop/HQKickStarter/Pods/Headers/Private/Bolts/BFAppLink.h
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:905:in `stat'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:905:in `symbolic_modes_to_i'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:926:in `fu_mode'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1000:in `block (2 levels) in chmod_R'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1460:in `preorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:998:in `block in chmod_R'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:997:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:997:in `chmod_R'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:117:in `block in prepare'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.38.2/lib/cocoapods/user_interface.rb:140:in `message'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:116:in `prepare'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:103:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.38.2/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.38.2/lib/cocoapods/command/project.rb:156:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.38.2/lib/cocoapods/command.rb:48:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.38.2/bin/pod:44:in `<top (required)>'
/usr/bin/pod:23:in `load'
/usr/bin/pod:23:in `<main>'
```

**Comment by segiddins** [comment]
This has been fixed in the 0.39 betas.

**Comment by marcelojunior7** [comment]
Ok, thanks.
